recipes/champ/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "champ" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "2e24d8fea58bf65bb13ec646cbb7404c4fc780ca4c07d5eb03c9aca9cb1609bb" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+  noarch: python
+
+
+requirements:
+
+  host:
+    - python
+    - igraph
+    - setuptools
+    - pip
+
+  run:
+    - python
+    - python-igraph
+    - matplotlib
+    - numpy >=1.13
+    - scipy >=0.19
+    - pandas
+    - h5py
+    - seaborn
+    - tqdm
+    - louvain
+    - scikit-learn
+    - future
+
+
+test:
+  imports:
+    - champ
+    - louvain
```

Comment:
This is coming from a dependency right? Or does `champ` effect how it is imported?
```suggestion
```

Reply:
champ doesn't affect import. removed from test.  thanks.